aws/logs_monitoring/cache.py

```diff
@@ -435,10 +440,141 @@ def get(self, log_group):
             # If the custom tag fetch env var is not set to true do not fetch
             if not self.should_fetch_tags():
                 logger.debug(
-                    "Not fetching custom tags because the env variable DD_FETCH_LAMBDA_TAGS is not set to true"
+                    "Not fetching custom tags because the env variable DD_FETCH_LOG_GROUP_TAGS is "
+                    "not set to true"
                 )
                 return []
             log_group_tags = get_log_group_tags(log_group) or []
             self.tags_by_id[log_group] = log_group_tags
 
         return log_group_tags
+
+
+#######################
+# Step Functions Tags #
+#######################
+
+
+class StepFunctionsTagsCache(LambdaTagsCache):
```

Comment:
Now that we have three caches, it might be a good idea to split these Cache classes out into their own files.